agenda/presentation/src/main/java/com/pronoidsoftware/agenda/presentation/overview/model/AgendaOverviewItemUi.kt

```diff
@@ -45,8 +47,8 @@ sealed class AgendaOverviewItemContents(
         override val id: String,
         override val title: String,
         override val description: String,
-        override val startDateTime: String,
-        val endDateTime: String,
+        override val startDateTime: UiText,
+        val endDateTime: UiText,
```

Comment:
If something is always a dynamic string (which I assume the formatted time is), I would personally consider it simpler to also just leave these fields a String. UiText is great to wrap around either just string resources or if a string sometimes originates out of a string resource and sometimes a normal string (like an API error message).